src/index.tsx

```diff
@@ -24,13 +24,13 @@ interface InPortalProps {
     children: React.ReactNode;
 }
 
-export const createPortalNode = <C extends Component<any>>(): PortalNode<C> => {
+export const createPortalNode = <C extends Component<any>>(tagName: string = 'div'): PortalNode<C> => {
     let initialProps = {} as ComponentProps<C>;
 
     let parent: Node | undefined;
     let lastPlaceholder: Node | undefined;
 
-    const portalNode = Object.assign(document.createElement('div'), {
+    const portalNode = Object.assign(document.createElement(tagName), {
```

Comment:
Viewing the story here in Firefox gives me a React warning:

> Warning: The tag `<text>` is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.

I think that's because SVG elements are supposed to be created explicitly with `document.createElementNS("http://www.w3.org/2000/svg", tagName)`, not with `document.createElement(tagName)`. See https://stackoverflow.com/questions/39457888/after-creating-a-text-element-with-javascript-why-cant-i-getbbox and https://github.com/facebook/react/issues/11899#issuecomment-355043661.

I'm not totally sure how React deals with this generally, but I'd guess that it has a list of SVG tag names, and calls the right method for each type. Happy to do that or something better, if you have any ideas?